src/SDKs/_metadata/datafactory_resource-manager.txt

```diff
@@ -1,11 +1,10 @@
-2017-11-22 21:57:17 UTC
+2017-12-07 02:41:02 UTC
 
 1) azure-rest-api-specs repository information
-GitHub user: Azure
+GitHub user: smilenow
```

Comment:
@smilenow why are you generating from your repo? This has to be generated from main azure fork and not yours?

Reply:
Because previously I heard from @gadongji , it could be OK no matter which branch is used to generated. But anyway, I will resolve the conflict and then re-generated by using the Azure fork branch.